.gitignore

```diff
@@ -1,8 +1,11 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
 *$py.class
 
+# Setting up of PyCharm
+.idea/
```

Comment:
I must keep IDE/Test editor stuff out of gitignore, use gitignore global localy for that.

Reply:
Ok! I understanding and I do it for the next PR!